src/client/automation/playback/click/index.js

```diff
@@ -222,21 +221,43 @@ export default class ClickAutomation {
         return focusAndSetSelection(elementForFocus, simulateFocus, this.caretPos);
     }
 
+    _getElementForClick (targetElement, topElement, targetElementParentNodes, topElementParentNodes) {
```

Comment:
@helen-dikareva @georgiy-abbasov  Hmmm. just read your discussion. You are right there are no reason to send this vars to a non-static method. But it will be difficult to understand that `this.eventArgs.element` is actually 'element before click event'.. Maybe it's better to do this method static?
```js
static _getElementForClick(mousedownElement, topElement, mousedownElementParents)
```